app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,43 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
```

Comment:
-[] kindly work on your login and signup user authentication when i try to signu to your deployed apps ,i am unable to signup successfully it keep showing error see below, the home and detail page should works as expected.

![Screenshot (34)](https://github.com/HenaryKhuraijam/budget-app/assets/92458236/61f2ecad-c3bc-4ebe-bb65-99bdc1b9cfd9)